Clarify file path syntax for `..` (#67)
metafacture/metafacture-documentation: docs/fix/Fix-User-Guide.md

#### File paths

> Some Fix functions can refer to files on your system. Paths are by default relative to the current working directory. Paths can always be stated in UNIX style - even when running Metafacture under Windows. If using Windows style you have to escape the backslashes (`C:\my\path` > `C:\\my\\path`). If you want to address files relative to the Fix file add `.` as directory (`./myFile`). To address files relative to the location of the executed Flux file, use the predefined `FLUX_DIR` variable, but you need to add the `*` to the `fix` command in the Flux (see above). You can also use `..` for the parent directory. At the beginning `..` is relative to the Fix file and can be used multiple times in order to relate to parent directories of parent directories (`../../myFile` relates to a file two parent directories up from the fix file.)  

Reviewer: ```suggestion
Some Fix functions can refer to files on your system. Paths are by default relative to the current working directory. Paths can always be stated in UNIX style - even when running Metafacture under Windows. If using Windows style, you have to escape the backslashes (`C:\my\path` > `C:\\my\\path`). If you want to address files relative to the Fix file, add `.` as directory (`./myFile`). To address files relative to the location of the executed Flux file, use the predefined `FLUX_DIR` variable, but you need to add the `*` to the `fix` command in the Flux (see above). You can also use `..` for the Fix file's parent directory (can be used multiple times in order to refer to parent directories of parent directories, e.g. `../../myFile` refers to a file two parent directories up from the Fix file).
```
